test/__snapshots__/Autocomplete.test.tsx.snap

```diff
@@ -141,12 +150,13 @@ Object {
               autocapitalize="none"
               autocomplete="off"
               class="MuiOutlinedInput-input MuiInputBase-input MuiInputBase-inputAdornedEnd MuiAutocomplete-input MuiAutocomplete-inputFocused css-nxo287-MuiInputBase-input-MuiOutlinedInput-input"
+              id=":r0:"
               name="hello"
               required=""
               role="combobox"
               spellcheck="false"
               type="text"
-              value=""
+              value="Hello"
```

Comment:
Should also look at this test too... maybe the snap was broken and is now fixed... but would have to investigate what the test is expecting here.

Reply:
If I swap the initial value to be the second data element, ie 'World' the value here will be 'World'.
If we are setting the initial value - this is what I would expect the value to be.